.changeset/stupid-items-greet.md

```diff
@@ -0,0 +1,10 @@
+---
+'@crowdstrike/glide-core': minor
+---
+
+- The default slot of Tab has been removed in favor of a `label` attribute.
+- Tab Group now throws if a Tab is added without a corresponding Tab Panel and vice versa.
+- `glide-core-tab-panel` has been renamed to `glide-core-tab-group-panel`.
+- The import of `@crowdstrike/glide-core/tab.panel.js` has been updated to `@crowdstrike/glide-core/tab-group.panel.js`.
+- `glide-core-tab` has been renamed to `glide-core-tab-group-tab`.
+- The import of `@crowdstrike/glide-core/tab.js` has been updated to `@crowdstrike/glide-core/tab-group.tab.js`.
```

Comment:
Which import are you referring to? I don't think people will know what this means. Tab's _import_ hasn't changed. Instead, Tab has been renamed and imports of it will have to be updated, no?

Reply:
https://github.com/CrowdStrike/glide-core/pull/1081/commits/9399f800a3f771881a6ed3c5b657068407840510